### DISCUSSION TITLE
Can I use kubevela CLI for transformation without a K8s cluster?

### DISCUSSION THREAD

**Discussion by nitagarwal** [kubevela/kubevela#6181] [Q&A]
We are planning to use kubevela CLI for transforming OAM into kubernetes manifests. 
We do not plan to use any community provided components and traits. 
The current instruction for kubevela require pointing to a kubeconfig file to even run the CLI.

Is it possible to run kubevela CLI without depending on a kubernetes cluster?

**Comment by wonderflow** [comment]
try adding the flag: `vela dry-run --offline`
